Dockerfile

```diff
@@ -1,21 +1,24 @@
 FROM continuumio/miniconda3
 
+RUN apt-get update && apt-get install -y libpng-dev
+
 RUN pip install Flask-Webpack
-RUN apt-get install -y libpng-dev
 
 RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - && \
     apt-get install -y nodejs && \
     npm install -g webpack && \
     npm install -g webpack-dev-server
 
-ADD ./ /flask-react/
+ADD ./ /fork-project/
 
 RUN npm install webpack -g
 RUN npm install webpack-dev-server -g
 
-RUN cd /flask-react && \
+RUN cd /fork-project && \
     npm install && \
-    webpack
+    webpack --config=webpack.prod.js
```

Comment:
I don't see a file `webpack.prod.js`. I don't think that this param will have any effect without it.